index.html

```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
                         <li class="icon">
                             <a href="https://github.com/JuanDiegoGuerra" rel="noopener" target="_blank">
                                 <img src="img/social media icons.png" alt="github link">
-                            </a></li>
+                            </a>
+                        </li>
                     </ul>
                 </div>
                </footer>
             </section>
     </div>
+    <div id="my-recent-works">
+        <h1 class="recent-works-tittle">My Recent <br> Works</h1>
+        <div class="menu-line-recent-works"></div>
+        <div class="project-1">
+            <div class="first_project_img_div">
+            <img src="img/default_project_img.JPG" alt="first project image" class="first_project_image">
+            </div>
+            <div class="first-project">
+              <h3>
+                Hello-Microverse-Project <br>
+                First Week
+              </h3>
+                <div class="languages-grid">                    
+                        <div class="lang-list l1"><strong>Html5</strong></div>
+                        <div class="lang-list l2"><strong>Css3</strong></div>
+                        <div class="lang-list l3"><strong>Git</strong></div>
+                        <div class="lang-list l4"><strong>Github</strong></div>
+                </div>
```

Comment:
- [ ] For this section, the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") is a list `ul` and each of them a `li`

## Instructions:
![image](https://user-images.githubusercontent.com/50990925/227673145-0c623600-cc8f-41b7-801f-ba8443f29116.png)